core/src/test/scala/com/rackspace/com/papi/components/checker/wadl/WADLCheckerSpec.scala

```diff
@@ -3289,6 +3301,88 @@ class WADLCheckerSpec extends BaseCheckerSpec with LogAssertions {
   }
 
 
+  scenario("The WADL contains PUT and POST operations accepting xml which must validate against an XSD (only grammar transform option is specified, warns disabled)") {
+    Given ("a WADL that contains multiple PUT and POST operation with XML that must validate against an XSD (only grammar transform option is specified, warns disabled)")
+    val inWADL =
+      <application xmlns="http://wadl.dev.java.net/2009/02">
+        <grammars>
+            <include href="src/test/resources/xsd/test-urlxsd.xsd"/>
+        </grammars>
+        <resources base="https://test.api.openstack.com">
+           <resource path="/a/b">
+               <method name="PUT">
+                  <request>
+                      <representation mediaType="application/xml"/>
+                      <representation mediaType="application/json"/>
+                  </request>
+               </method>
+               <method name="POST">
+                  <request>
+                      <representation mediaType="application/xml"/>
+                  </request>
+               </method>
+           </resource>
+           <resource path="/c">
+               <method name="POST">
+                  <request>
+                      <representation mediaType="application/json"/>
+                  </request>
+               </method>
+               <method name="GET"/>
+           </resource>
+           <resource path="/any">
+              <method name="POST">
+                 <request>
+                    <representation mediaType="*/*"/>
+                 </request>
+              </method>
+           </resource>
+           <resource path="/text">
+              <method name="POST">
+                 <request>
+                    <representation mediaType="text/*"/>
+                 </request>
+              </method>
+           </resource>
+           <resource path="/v">
+              <method name="POST">
+                 <request>
+                    <representation mediaType="text/plain;charset=UTF8"/>
+                 </request>
+              </method>
+           </resource>
+        </resources>
+    </application>
+    register("test://app/src/test/resources/xsd/test-urlxsd.xsd",
+             XML.loadFile("src/test/resources/xsd/test-urlxsd.xsd"))
+    When("the wadl is translated")
+    val config = TestConfig(false, false, false, false, false, 1, false, true)
+    config.enableWarnHeaders = false
+    val checker = builder.build (inWADL, config)
+    reqTypeAssertions(checker)
+    wellFormedAssertions(checker)
+    xsdAssertions(checker)
+    And("The following assertions should also hold:")
```

Comment:
Yes, I'm really starting to regret that pattern here.  Since this doesn't change the assertions that are checked I'll save these type of changes for #109.